### ISSUE TITLE
Range Function is Incorrect

### ISSUE THREAD

**Issue by garv-shah** [Decimation/Nspire-Library#1]
The range function doesn't seem to output the correct range, for example `(x-1)^2 + 2` outputs all real numbers

**Comment by Decimation** [comment]
You're correct, I believe I wrote this function back in high school pre-calculus where I was taught that a function's range can be derived from the domain of its inverse. The range function implements this by returning `domain(1/f,x)` (last I checked)
